app/frontend/src/views/About.vue

```diff
@@ -93,9 +70,14 @@
 
 <script>
 import { mapGetters } from 'vuex';
+import BaseDisclaimer from '@/components/base/BaseDisclaimer.vue';
```

Comment:
Discussed above that wouldn't have this in `base` but FYI for the future that components in the Base folder don't need to be manually imported. The intent with those is they're small frequent components that you can just reference in any SFC without needing to import each time.

Reply:
ok. good to know. ty